### PR TITLE
fix(test) segmented tests with transient failures

### PR DESCRIPTION
**Pull request by dshekhar95**
## Problem
Transient issues running restore/minio/multi-tenancy tests
 

## Solution
#skip these tests in full suite run, and then run them individually after

### CODE REVIEW

File: .github/workflows/ci-dgraph-tests.yml [hunk 1]
```diff
@@ -60,9 +60,15 @@ jobs:
           # build the test binary
           cd t; go build .
           # run the tests
-          ./t --coverage=true
+          ./t --coverage=true --skip=restore,multi-tenancy,minio
```
Comment (skrdgraph) [inline .github/workflows/ci-dgraph-tests.yml:63]: you need FQP (fully qualified path)

File: .github/workflows/ci-dgraph-tests.yml [hunk 2]
```diff
@@ -60,9 +60,15 @@ jobs:
...
           # clean up docker containers after test execution
           ./t -r
+          ./t --pkg=restore
```
Comment (skrdgraph) [inline .github/workflows/ci-dgraph-tests.yml:66]: change to FQP (fully qualified path)

File: .github/workflows/ci-dgraph-tests.yml [hunk 3]
```diff
@@ -60,9 +60,15 @@ jobs:
...
+          ./t -r
+          ./t --pkg=multi-tenancy
```
Comment (skrdgraph) [inline .github/workflows/ci-dgraph-tests.yml:68]: change to FQP (fully qualified path)

File: .github/workflows/ci-dgraph-tests.yml [hunk 4]
```diff
@@ -60,9 +60,15 @@ jobs:
...
+          ./t -r
+          ./t --pkg=minio
```
Comment (skrdgraph) [inline .github/workflows/ci-dgraph-tests.yml:70]: change to FQP (fully qualified path)

**Review by skrdgraph — COMMENTED** [review]
change to FQP (fully qualified path)

File: .github/workflows/ci-dgraph-tests.yml (hunk 1 above)
Comment (skrdgraph) [reply .github/workflows/ci-dgraph-tests.yml]: fixed in https://github.com/dgraph-io/dgraph/pull/8351/commits/3d9f7ff0ab02b4e057598a3d7139b1a4f356650f

File: .github/workflows/ci-dgraph-tests.yml (hunk 2 above)
Comment (skrdgraph) [reply .github/workflows/ci-dgraph-tests.yml]: fixed in https://github.com/dgraph-io/dgraph/pull/8351/commits/3d9f7ff0ab02b4e057598a3d7139b1a4f356650f

File: .github/workflows/ci-dgraph-tests.yml (hunk 3 above)
Comment (skrdgraph) [reply .github/workflows/ci-dgraph-tests.yml]: fixed in https://github.com/dgraph-io/dgraph/pull/8351/commits/3d9f7ff0ab02b4e057598a3d7139b1a4f356650f

File: .github/workflows/ci-dgraph-tests.yml (hunk 4 above)
Comment (skrdgraph) [reply .github/workflows/ci-dgraph-tests.yml]: fixed in https://github.com/dgraph-io/dgraph/pull/8351/commits/3d9f7ff0ab02b4e057598a3d7139b1a4f356650f

**Comment by skrdgraph** [comment]
@joshua-goldstein could you re-review this one please? I modified the package to use FQPs

**Review by meghalims — APPROVED** [review]
After skip list are using QFP, the tests are skipped as expected and the run is looking good. I am approving it

**Pull request merged**